sdk/Trace/BatchSpanProcessor.php

```diff
@@ -85,7 +84,7 @@ public function onEnd(API\Span $span): void
         if ($this->bufferReachedExportLimit() && $this->enoughTimeHasPassed()) {
             $this->exporter->export($this->queue);
             $this->queue = [];
-            $this->lastExportTimestamp = (int) \round((float) $this->clock->millitime());
+            $this->lastExportTimestamp = $this->clock->timestamp();
```

Comment:
I think this is supposed to be `now()`?
```suggestion
            $this->lastExportTimestamp = $this->clock->now();
```